### ISSUE TITLE
Typings are not available

### ISSUE THREAD

**Issue by Lonli-Lokli** [okta/okta-signin-widget#2385]
### I'm submitting a
  - [x] bug report
  - [ ] feature request

### Background info
I am getting an error in Typescript porject when trying to use v6 of this library
`import * as OktaSignIn from '@okta/okta-signin-widget';
`
Could not find a declaration file for module '@okta/okta-signin-widget'. 'D:/gitlab/cv-app/frontend/cv-ui/node_modules/@okta/okta-signin-widget/dist/js/okta-sign-in.entry.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/okta__okta-signin-widget` if it exists or add a new declaration (.d.ts) file containing `declare module '@okta/okta-signin-widget';`ts

### Expected behavior
No error 

I am aware of workaround but it's better to have correct typings
```
// @ts-ignore
import OktaSignIn from '@okta/okta-signin-widget';
```

### Your environment

  - Okta Sign-In Widget Version: 6
  - Browser: Chrome
  - OS: Win11
  - Language: UK

**Comment by arvindkrishnakumar-okta** [comment]
@Lonli-Lokli Thanks for posting! 

Someone from our team will review this soon. cc: @aarongranick-okta @shuowu-okta

**Comment by shuowu** [comment]
@Lonli-Lokli Types are still not supported in v6, please keep using the `// @ts-ignore` workaround before official support is released.

**Comment by aarongranick-okta** [comment]
@Lonli-Lokli types were published in 6.3.0. I recommend using the latest version for the latest type definitions and fixes.
